presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -3,7 +3,8 @@
 # so we canonicalize them.
 function canonical_path() {
     # make the path start with '/' or './'. Otherwise the result for 'foo.txt' is an absolute path.
-    local PATH=`[[ $1 = /* ]] && echo "$1" || echo "./${1#./}"`
+    local PATH_REFERENCE=$1
+    local PATH=`[[ ${!PATH_REFERENCE} = /* ]] && echo "${!PATH_REFERENCE}" || echo "./${!PATH_REFERENCE#./}"`
```

Comment:
In one of my PRs David mentioned the `$( ... )` syntax is preferred to ``` `...` ```. I agree that it's more readable with `$( )`